restli-server/src/test/java/com/linkedin/restli/server/TestCollectionResult.java

```diff
@@ -0,0 +1,111 @@
+package com.linkedin.restli.server;
```

Comment:
nit: add license to the top of all files with the current year (see any other file and copy-paste)

Reply:
Sure, fixed.